### ISSUE TITLE
brucella infectious disposition and brucella zoonotic disposition

### ISSUE THREAD

**Issue by PhiBabs935** [biomedontology/idobru#3]
Brucella infectious disposition =def. An infectious disposition that is the disposition of Brucella to be transmitted to a host and establish a Brucella infectious disorder.
Brucella zoonotic disposition = def. An infectious disposition that is the disposition of Brucella to be transmitted from an infected non-human host to a human host.

Asiyah, you noted previously (in your comments on the Re-engineering IDOBRU doc) that the above definitions of 'brucella infectious disposition' and 'brucella zoonotic disposition' lead to multiple parentage--'Brucella zoonotic disposition' ends up a child of 'Brucella infectious disposition'. 

I think this issue can be avoided if we leverage the infectious disposition subclass 'communicability', which is disjoint with 'zoonotic disposition.'

I suggest that we dispense with the class 'Brucella infectious disposition' and instead add the class 'Brucella communicability' and define it as: "An infectious disposition that is the disposition of Brucella to be transmitted directly from one organism to another of the same Species by horizontal transmission'.
-In this case, 'Brucella zoonotic disposition' obviously won't be a child of 'Brucella communicability'

Incidentally, I was looking back from Lindsay's comments on an earlier draft of the IDO paper. She seems to think that we don't really need terms like Brucella communicability or Brucella infectious disposition. She thinks it might be enough to simply add associated axioms such as "Brucella bears an infectious disposition"
I am not sure that works though: an IDO:infectious disposition is (partly) a disposition to establish an infectious disorder. But Brucella has something more specific, a disposition to establish a _Brucella_ _infectious disorder_ (which we specify in our previous definition of Brucella infectious disposition). The axiom that invokes only 'infectious disposition' doesn't capture this fact.

**Comment by PhiBabs935** [comment]
Although now I realize it's not that simple...as our new definition of _	Brucella pathogen role_ made use of _Brucella infectious disposition_

If we go with the fix suggested above, maybe just define _Brucella pathogen role_ as "A pathogen role borne by Brucella bacterium when contained in a host in which its infectious disposition can be realized." And maybe add in a comment that the infectious disposition can be either a Brucella communicability or a Brucella zoonotic disposition...or something along those lines.

**Comment by linikujp** [comment]
Are you sure if the Brucella only have two transmission route:
between the same species and from non-human to human?
If above believe is true, then we can use your strategy. We will need @yongqunh to weigh in here.

**Comment by PhiBabs935** [comment]
I am not sure of that actually. The original changes suggested for these terms came after Oliver pointed out in his comments that there was more than one transmission route. I agree that we should confer with him.

**Comment by yongqunh** [comment]
I think that 'zoonotic disposition' is quite understandable for epidemiologists and infectious disease researchers. 'zoonotic' means transmission between humans and animals. 'communicability' may not reflect such meaning. It is important to keep this in mind.

**Comment by linikujp** [comment]
Meeting on 05/19/2020 John B. Shane B, Oliver H and Asiyah L.
Oliver suggested to use transmissibility , the disposition is realized in a transmission process. A disposition to undergo a transmission process. (from a non-host or source to a host, or from a host to another host) -- IDO may need to improve the textual definition for transmissibility disposition, or add a comment. 
Brucella infectious disposition -- between host and pathogen
Brucella zoonotic disposition -- between host and another host

More research on Brucella transmission routes.

**Comment by PhiBabs935** [comment]
So, to follow up on our discussion. 

The issue is that as long as we have both 'Brucella infectious disposition' and 'Brucella zoonotic disposition', the latter class is both a child of (IDO) 'zoonotic disposition' and a child of  'brucella infectious disposition'. 

So, if we follow Oliver's suggestion and add 'transmissibility disposition' to IDOBRU would the idea be to remove 'Brucella infectious disposition' and instead add a Brucella specific subclass of 'transmissibility disposition'? Such as:
'Brucella transmissibility disposition' Would this avoid the issue though? Or would 'Brucella zoonotic disposition' also end up as a subclass of this? If we define it as a kind of infectious disposition, then it seems that the original problem will just crop up again.

**Comment by yongqunh** [comment]
I don't think that 'Brucella zoonotic disposition' is a child of 'brucella infectious disposition', unless we think .'zoonotic disposition' is a child of 'infectious disposition'. 
infection is for pathogen to infect host. zoonotic disposition is for pathogen able to infect both human and animal. If we think it is correct that 'zoonotic disposition' is a child of 'infectious disposition', it may be fine depending on how we define it.

**Comment by PhiBabs935** [comment]
In IDO, 'zoonotic disposition' is a child of 'infectious disposition'.
If that's right about zoonosis, we would have to discuss with Lindsay and Alex Diehl then, because in IDO they restrict 'zoonotic disposition' to non-human to human transmission: "An infectious disposition that is the disposition to be transmitted from an infected, non-human host to a human host."
You could define a brucella disposition that doesn't have 'brucella zoonotic disposition' as a child, but I don't think it would be right to call it 'brucella infectious disposition' because that label seems to imply a broader class that subsumes different kinds of brucella transmissibility. 

I guess I need to make sure I have the science straight too here. Are all Brucella zoonotic? Are there any Brucella that are unable to infect humans?

**Comment by yongqunh** [comment]
yes. there are Brucella strains that are unable to infect humans.

**Comment by yongqunh** [comment]
I am not sure whether it is correct to say that 'zoonotic disposition' is a child of 'infectious disposition'. We will need to see the definitions. One is for pathogen infecting host. one is for transmission from animals to humans.

**Comment by PhiBabs935** [comment]
If not a child of infectious disposition, then what alternatively could it be a child of in IDO? Are you suggesting that it should be a subclass of 'transmissibility disposition' instead?

**Comment by yongqunh** [comment]
Indeed, I would suggest that it be a subclass of 'transmissibility disposition'.

**Comment by PhiBabs935** [comment]
I see. If that works it would block the whole multiple parentage issue. So are there cases of zoonotic transmissions of a pathogen that don't involve infection?

**Comment by yongqunh** [comment]
They are different scopes. The answer depends on how you define infection. 
I think you are really talk about the comparison between transmission and infection. Zoonotic transmission is a subclass of transmission. 

Infection is for pathogen to infect a host. transmission is for transmission of a disease from one source to a host.

**Comment by yongqunh** [comment]
By the way, I don't like IDO definition of infection as a material (or disorder as a subclass of material). It's a process, not a material.

**Comment by yongqunh** [comment]
So we can have:
    'transmissibility disposition'.
              'zoonotic transmissibility disposition'.

**Comment by yongqunh** [comment]
"While all communicable diseases are infectious, not all infections are communicable. Tetanus, for example, can cause an infection, but a person with tetanus can't spread it to other people." 
https://www.verywellhealth.com/the-difference-between-communicable-and-infectious-diseases-4151985

**Comment by yongqunh** [comment]
"Here’s the big takeaway. All contagious diseases are infectious, but not all infectious diseases are contagious. Contagious diseases are infectious diseases that are easily spread through contact with other people."  https://www.dictionary.com/e/contagious-vs-infectious-the-difference-can-be-important/

**Comment by PhiBabs935** [comment]
At some point we will all have to confer with the rest of the IDO team (e.g. Alex. Lindsay). I am not in control of the IDO hierarchy myself. 

I think the stance of the IDO team is that strictly speaking, diseases are not transmitted--according to IDO, pathogens are all that are transmitted and then cause diseases once they cause a disorder in the host. I think Lindsay would argue that to say that a disease is transmitted from one person to another is a loose way of speaking. Consider the OGMS/IDO model of disease as a disposition to undergo pathological processes. An individual pathogen can move from one host to another host - into me, for instance - but technically the disposition that I have to undergo pathological processes (as a result of being infected by the transmitted pathogen) is not something that was transmitted into me from somewhere else. Of course, the previous host may have had the same _type_ of disposition/disease. But I think the IDO team (actually, TRANS from which the term is imported) is using 'transmission' in a very strict sense to apply to pathogens only. But this is something we should all discuss in a call.

The IDO Core team was pretty adamant about retaining the IDO definition for infection. Plus the disorder as a material basis of a disease is pretty fundamental to OGMS and IDO. That said, Lindsay agrees that there is a process of infection and is considering adding such a process in addition to the material entity.

I probably won't reply anymore for now, I have a lot to think about!
